.changeset/happy-glasses-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
```

Comment:
you also added the `optionalLabel` prop too

Reply:
True, thanks for catching that!